lib/cequel/record/dirty.rb

```diff
@@ -47,8 +47,7 @@ def map(name, *)
       def save(options = {})
         super.tap do |success|
           if success
-            @previously_changed = changes
-            @changed_attributes.clear
+            changes_applied
```

Comment:
This change is also interesting, why wasn't `ActiveModel::Dirty`'s `changes_applied` not used here instead?